013-quarkus-oidc-restclient/src/test/java/io/quarkus/qe/NativeTokenPropagationPingPongResourceIT.java

```diff
@@ -0,0 +1,10 @@
+package io.quarkus.qe;
+
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.test.junit.NativeImageTest;
+
+@Disabled("Can't use both OIDC Client Filter and OIDC Token extensions. Reported in https://github.com/quarkusio/quarkus/issues/14466")
```

Comment:
TODO again. It shows nicely in IDEs/editors :)

Reply:
added